verticapy/connection/connect.py

```diff
@@ -114,6 +148,13 @@ def set_connection(conn: Connection) -> None:
 def close_connection() -> None:
     """
     Closes the connection to the database.
+
+    Example
+    -------
+    .. code-block:: python
+
+        from verticapy.connection import close_connection
```

Comment:
add a space in between the import and the function.

Add also a line before:
1. Let's close all the current connections.
.. warning:: When you close the connection, your session will terminate, and all temporary elements will be automatically dropped.